src/stories/dropdown.stories.js

```diff
@@ -18,7 +18,7 @@ export const Basic = () => html`
       >
         Dropdown button
       </button>
-      <ul class="dropdown-menu">
+      <ul class="dropdown-menu" id="dropdown-menu">
```

Comment:
These HTML elements with the class `no-wc` are not actually included in the custom element. They get filtered out by Javascript (see line 27-28 of Dropdown.js). The reason they exist is to support browsers that don't have Javascript enabled. 

The elements that actually get included start at line 29 are what get included in custom element shadow DOM. Those are the elements where we should set the IDs.

So, for example, line 29-38 should be:
```html
    <cod-button
      id="dropdownControl"
      data-label="Dropdown button"
      data-label="Dropdown button"
      data-background-color="primary"
      data-background-color="primary"
      data-primary="true"
      data-primary="true"
      data-img-alt=""
      data-img-alt=""
      data-icon=""
      data-icon=""
      data-bs-toggle="dropdown"
      data-bs-toggle="dropdown"
      aria-expanded="false"
      aria-expanded="false"
      data-extra-classes="dropdown-toggle"
      data-extra-classes="dropdown-toggle"
    ></cod-button>
```

Reply:
Then we would want to similarly set an ID on the dropdown menu element as well.